pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -165,7 +168,11 @@ func TestGetRemoteEntrypoint(t *testing.T) {
 	}))
 	defer server.Close()
 	image := path.Join(strings.TrimPrefix(server.URL, "http://"), "image:latest")
-	ep, err := entrypoint.GetRemoteEntrypoint(entrypoint.NewCache(), image)
+	entrypointCache, err := NewCache()
+	if err != nil {
+		t.Errorf("couldn't create new entrypoint cache: %v", err)
```

Comment:
I think it should be `t.Fatalf` here. We want the test to fail if object creation fails here.